.github/workflows/ci.yml

```diff
@@ -3,11 +3,8 @@ name: Java CI
 on: [push]
 
 jobs:
-  build:
+  build-cassandra-3_11:
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        backend: [ "cassandra-3.11", "cassandra-4.0", "dse-6.8" ]
```

Comment:
Are multiple jobs that much different from a matrix job? I'd imagine the execution is going to be the same... it certainly looks very similar in the build console :thinking:

Reply:
https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix

> A matrix allows you to create multiple jobs by performing variable substitution in a single job definition

So I guess if the matrix is creating multiple jobs then, no, multiple jobs and matrix are not much different. Maybe the improvement I was seeing was from using the cache?